tests/robot-tests/tests/admin_and_public/bau/archive_publication.robot

```diff
@@ -0,0 +1,346 @@
+    user create test release via api    ${PUB_ID_SUPERSEDE}    FY    2000
+
+Set archive-publication to be superseded by superseding-publication
+    user navigates to admin dashboard
+    user opens accordion section    ${PUBLICATION_NAME_ARCHIVE}
+    ${ARCHIVE_ACCORDION}=    user gets accordion section content element    ${PUBLICATION_NAME_ARCHIVE}
+    user clicks link    Manage publication    ${ARCHIVE_ACCORDION}
+
+    user waits until h1 is visible    Manage publication    %{WAIT_SMALL}
+    user waits until page contains element    id:publicationForm-supersede
+
+    user chooses select option    id:publicationForm-supersededById    ${PUBLICATION_NAME_SUPERSEDE}
+
+    user clicks button    Save publication
+    user waits until modal is visible    Confirm publication changes
+    user clicks button    Confirm
+    user waits until modal is not visible    Confirm publication changes
+
+Validate archive warning is on Admin dashboard for archive-publication release
+    ${accordion}=    user opens publication on the admin dashboard    ${PUBLICATION_NAME_ARCHIVE}
+    user checks element should contain    ${accordion}
+    ...    This publication will be archived when its superseding publication has a live release published.
+
+Check public site, that archive-publication still appears correctly
+    user navigates to find statistics page on public frontend
+
+    user waits until page contains accordion section    %{TEST_THEME_NAME}
+    user opens accordion section    %{TEST_THEME_NAME}
+    user waits until accordion section contains text    %{TEST_THEME_NAME}    %{TEST_TOPIC_NAME}
+
+    user opens details dropdown    %{TEST_TOPIC_NAME}
+    user waits until details dropdown contains publication    %{TEST_TOPIC_NAME}    ${PUBLICATION_NAME_ARCHIVE}
+    user checks publication bullet contains link    ${PUBLICATION_NAME_ARCHIVE}    View statistics and data
+
+    user clicks element    testid:View stats link for ${PUBLICATION_NAME_ARCHIVE}
+
+    user waits until h1 is visible    ${PUBLICATION_NAME_ARCHIVE}    %{WAIT_MEDIUM}
+    user waits until page contains    This is the latest data
+    ${PUB_ARCHIVE_URL}=    user gets url
+    set global variable    ${PUB_ARCHIVE_URL}
+
+Check Data tables and Data catalogue pages, that archive-publication still appears correctly
+    user navigates to data tables page on public frontend
+
+    user opens details dropdown    %{TEST_THEME_NAME}
+    user opens details dropdown    %{TEST_TOPIC_NAME}
+
+    user checks page does not contain    ${PUBLICATION_NAME_SUPERSEDE}
+
+    user clicks radio    ${PUBLICATION_NAME_ARCHIVE}
+    user clicks element    id:publicationForm-submit
+    user waits until table tool wizard step is available    2    Choose a subject
+    user checks previous table tool step contains    1    Publication    ${PUBLICATION_NAME_ARCHIVE}
+
+    user checks page contains    ${SUBJECT_NAME_ARCHIVE}
```

Comment:
Can we be more specific with this selector?

Reply:
There is no way that this is going to be detected on a different part of the page, so I don't see any issue with checking the whole page here.